README.md

```diff
@@ -1,4 +1,12 @@
-# MuseScore: Music notation and composition software
+<p align="center">
+   <img src="mscore/data/musescore_logo_full.png"></img><br/>
+   Music notation and composition software
+</p>
+
+<p align="center">
+	<a href="https://travis-ci.org/musescore/MuseScore"><img src="https://secure.travis-ci.org/musescore/MuseScore.svg" alt="Travis CL"></a>
+   <a href="https://www.gnu.org/licenses/old-licenses/gpl-2.0.html"><img src="https://img.shields.io/badge/License-GPL%20v2-blue.svg" alt="License: GPL v2"></a>
```

Comment:
And use `![License: GPL v](https://img.shields.io/badge/License-GPL%20v2-blue.svg)](https://www.gnu.org/licenses/old-licenses/gpl-2.0.html)`